service/src/java/org/apache/hive/service/cli/CLIService.java

```diff
@@ -149,6 +149,10 @@ public UserGroupInformation getHttpUGI() {
     return this.httpUGI;
   }
 
+  public HiveServer2 getHiveServer2() {
```

Comment:
instead of giving access to the HiveServer2 object: how about a method like `getWebServerUrl()` ?

Reply:
yes, thank you for your good advice, @kgyrtkirk!